modules/virt-about-runstrategies-vms.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+[id="virt-about-runstrategies-vms_{context}"]
+= About RunStrategies for virtual machines
+
+A `RunStrategy` for virtual machines determines a virtual machine instance (VMI)'s behavior, depending on a series of conditions. The `spec.runStrategy` setting exists in the virtual machine configuration process as an alternative to the `spec.running` setting.
+The `spec.runStrategy` setting allows greater flexibility for how VMIs are created and managed, in contrast to the `spec.running` setting with only `true` or `false` responses. However, the two settings are mutually exclusive. Only either `spec.running` or `spec.runStrategy` can be used. An error occurs if both are used.
+
+There are four defined RunStrategies.
+
+* Always: A VMI is always present when a virtual machine is created. A new VMI is created if the original stops for any reason, which is the same behavior as `spec.running: true`.
+* RerunOnFailure: A VMI is re-created if the previous instance fails due to an error. The instance is not re-created if the virtual machine stops successfully, such as when it shuts down.
+* Manual: The `start`, `stop`, and `restart` virtctl client commands can be used to control the VMI's state and existence.
+* Halted: No VMI is present when a virtual machine is created, which is the same behavior as `spec.running: false`.
+
+Different combinations of the `start`, `stop` and `restart` virtctl commands affect which `RunStrategy` is used.
+
+.`RunStrategies`
+
+[caption=]
+.RunStrategies
+|===
+|RunStrategy |start |stop |restart
+
+|Always
+|-
+|Halted
+|Always
+
+|RerunOnFailure
+|-
+|Halted
+|RerunOnFailure
+
+|Manual
+|Manual
+|Manual
+|Manual
+
+|Halted
+|Always
+|-
+|-
+|===
+
+[NOTE]
+====
+In {VirtProductName} clusters installed using installer-provisioned installation, when a node fails the MachineHealthCheck and becomes unavailable to the cluster, VMs with a RunStrategy of `Always` or `RerunOnFailure` are rescheduled on a new node.
+====
+
+[caption=]
+.Example snippet of virtual machine configuration with RunStrategy
```

Comment:
I don't think that "snippet" is necessary when you're already saying "example".

Also, I'm not sure what the [caption=] is for, but I assume you don't need it, so I'd remove it.